src/main/java/com/deavensoft/paketomat/center/CenterController.java

```diff
@@ -58,16 +59,15 @@ public int savePackage(@RequestBody Package newPackage) throws IOException, Pake
         } else{
             String messages = "User exist";
             log.info(messages);
-
             EmailDetails emailDetails = new EmailDetails();
             Map<String, Object> model = new HashMap<>();
             emailDetails.setMsgBody("Package arrived at the distribution center");
             emailDetails.setRecipient(user.get().getEmail());
             emailDetails.setSubject("test");
             model.put("msgBody", emailDetails.getMsgBody());
-            emailService.sendMailWithTemplate(emailDetails, model);
-            dispatcherService.delieverPackage(newPackage);
-            log.info("Package is ready to be dispatched");
+            //emailService.sendMailWithTemplate(emailDetails, model);
+            //dispatcherService.delieverPackage(newPackage);
+            //log.info("Package is ready to be dispatched");
```

Comment:
Unused code should be deleted and can be retrieved from source control history if required.

- [ ] Delete unused code

Reply:
Private fields are assigned but never accessed. Fix it.

Reply:
We deleted it